### PR TITLE
MWOR - New GNSS site

### PR DESCRIPTION
**Pull request by EvanCGNS**
Antenna already existed in antennas.csv.

Changes to:
assets/receivers.csv
environment/visibility.csv
install/antennas.csv
install/firmware.csv
install/receivers.csv
install/sessions.csv
network/marks.csv
network/monuments.csv

Related to: https://github.com/GeoNet/tickets/issues/17092

### CODE REVIEW

File: network/monuments.csv [hunk 1]
```diff
@@ -99,6 +99,7 @@ MTJO,50204M001,Forced Centering,Pillar,-2.4,Reinforced Concrete,2,2000-07-31T00:
 MTPR,,Forced Centering,Shallow Rod / Braced Antenna Mount,-1.262,Stainless Steel Rods,2.5,2010-01-27T00:00:00Z,9999-01-01T00:00:00Z,,
 MTQN,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.51,Stainless Steel Rods,10,2008-06-30T00:00:00Z,9999-01-01T00:00:00Z,Greywacke,Mt Quoin In South Tararuas
 MUL1,,Unmarked Mark,Unknown,-0.797,Unknown,0,2016-11-16T00:00:00Z,2018-01-15T00:00:00Z,,
+MWOR,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.50,Stainless Steel Rods,10,2024-12-04T00:00:00Z,9999-01-01T00:00:00Z,Breccia,"Laharic breccia of andesite cobbles in a muddy matrix, and conglomerate and sand with interbedded peat."
```
Comment (aleks-spes) [inline network/monuments.csv:102]: Hey @EvanCGNS ! Where did you get these descriptions of Bedrock and Geology?

Comment (EvanCGNS) [reply network/monuments.csv:102]: @aleks-spes used this map! https://www.gns.cri.nz/data-and-resources/geological-map-of-new-zealand/

Comment (aleks-spes) [reply network/monuments.csv:102]: Cool! So in this case you can use this map to put more info regarding geology into the PR: https://github.com/GeoNet/delta/pull/2310

**Review by aleks-spes — APPROVED** [review]

**Comment by EvanCGNS** [comment]
Hi @gva6500 can you take a look at this PR as @CBurton90 is away? Cheers!

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**